apis/v1alpha1/scalable_target.go

```diff
@@ -22,16 +22,166 @@ import (
 
 // ScalableTargetSpec defines the desired state of ScalableTarget
 type ScalableTargetSpec struct {
+	// The maximum value that you plan to scale out to. When a scaling policy is
```

Comment:
this API and scheduled action to not return an Arn, how does customer know if create succeeded.

 there are other apis which do not have arn, check if they removed arn from crd, and what they have done for status

Reply:
TBD.

Reply:
As discussed, added ARN for policy. CreationTime for ScalableTarget needs changed to the code generator so made change as of now.